Dockerfile

```diff
@@ -63,7 +63,8 @@ RUN \
 COPY /bin /opt/opal/bin
 COPY /data /opt/opal/data
 
-RUN adduser --system --home $OPAL_HOME --no-create-home --disabled-password opal; \
+RUN groupadd --system --gid 10041 opal && \
+    useradd --system --home $OPAL_HOME --no-create-home --uid 10041 --gid opal opal; \
```

Comment:
Did you chose 10041 arbitrarily? (any other number not already assigned and steadily applied would have worked)

Reply:
Definitely arbitrarily, I wanted it to be > 10.000 to avoid host-uid collisions (see also https://docs.prismacloud.io/en/enterprise-edition/policy-reference/kubernetes-policies/kubernetes-policy-index/bc-k8s-37)

If you pretend the second 0 is a P, the 4 is an A and the 1 is an L, you get close to OPAL...